index.html

```diff
@@ -214,35 +220,22 @@ <h2>
 did:web:w3c-ccg.github.io:user:alice
 
 did:web:example.com%3A3000
-        </pre>
-      </section>
 
-      <section>
-        <h2>
-JSON-LD context definition
-        </h2>
-        <p>
-The definition of the Web DID JSON-LD context is:
-        </p>
+did:web:example.com%3A3000:bob%3Awork
 
-        <pre title="Web DID method JSON-LD context">
-{
-    "@context":
-    {
-        "ethereumAddress": "https://github.com/uport-project/ethr-did-resolver#ethereumAddress",
-        "Secp256k1VerificationKey2018": "https://github.com/uport-project/ethr-did-resolver#Secp256k1VerificationKey2018",
-        "Secp256k1SignatureAuthentication2018": "https://github.com/uport-project/ethr-did-resolver#Secp256k1VerificationKey2018",
-    }
-}
+did:web:example.com:christine:111.json
```

Comment:
can we not mix these file name based variations into this PR?

Too many changes at once, IMO.

Reply:
A reduced PR with a clean history appears at #47.